src/index.js

```diff
@@ -29,12 +26,28 @@ const defaultOptions = {
 
 const SNAPSHOT_TITLE = 'Snapshot Diff:\n';
 
+const identity = value => value;
+let serializers = [reactSerializer];
+
 const snapshotDiff = (valueA: any, valueB: any, options?: Options): string => {
   let difference;
   const mergedOptions = { ...defaultOptions, ...options };
 
-  if (isReactComponent(valueA) && isReactComponent(valueB)) {
-    difference = diffReactComponents(valueA, valueB, mergedOptions);
+  const matchingSerializer = serializers.find(
+    ({ test }) => test(valueA) && test(valueB)
```

Comment:
Since we're now applying serializers one-by-one until we find a proper one, it would be nice to document that the order matters.